templates/paloalto_panos_show_arp_all.template

```diff
@@ -1,14 +1,15 @@
 Value INTERFACE (\S+)
 Value IP (\S+)
-Value MAC ([0-9a-fA-F:]+)
+Value MAC ([0-9A-Fa-f:]+|incomplete)
 Value PORT (\S+)
 Value STATUS (\S+)
 Value TTL (\d+)
 
 Start
+  ^interface\s+ip address\s+hw address\s+port\s+status\s+ttl
```

Comment:
L9 add the **-> Start_record** to the end of it, and the move L10 to the first line under **Start_record**. We're looking to start recording if the headers are correct so we're looking for the correct data.